README.md

```diff
@@ -8,39 +8,55 @@
 - [Maki](https://github.com/appleseedlab/maki)
 
 ## Instructions for using MerC
-1. Download and extract source for target program.
+### 1. Download and extract source for target program.
 
-	For example, if running on bc:
-	wget https://mirrors.kernel.org/gnu/bc/bc-1.07.tar.gz
-	tar -xzf bc-1.07.tar.gz
-	Ensure you have the dependencies:
-	apt-get install ed
-	apt-get install texinfo
+For example, if running on bc:
+```
+wget https://mirrors.kernel.org/gnu/bc/bc-1.07.tar.gz
+tar -xzf bc-1.07.tar.gz
+```
+Ensure you have the dependencies:
+```
+apt-get install ed
+apt-get install texinfo
+```
+### 2. Use Bear or cmake to intercept the build process to create a `compile_commands.json`
 
-2. Use Bear or cmake to intercept the build process to create a `compile_commands.json`
+For example, if running on bc, from within the bc directory: 
+```
+bear -- ./configure
+bear -- make
+```
 
-  For example, if running on bc, from within the bc directory:
-  bear -- ./configure
-  bear -- make
+You should now have a `compile_commands.json` in the directory of your target program.
 
-  You should now have a `compile_commands.json` in the directory of your target program.
+### 3. Run `run_maki_on_compile_commands.py` from MerC 
 
-3. Run `run_maki_on_compile_commands.py` from MerC
-  The format for running the script is as follows:
-  `python3 run_maki_on_compile_commands.py -p <path to maki plugin> -i <target program source directory> -c <path to compile_commands.json> -o <path to output maki analysis file. Default is analysis.maki> -j <number of threads, default is number of CPUs on system> -v <verbose>`
+The format for running the script is as follows:
+
+```
+python3 run_maki_on_compile_commands.py -p <path to maki plugin> -i <target program source directory> -c <path to compile_commands.json> -o <path to output maki analysis file, default is analysis.maki> -j <number of threads, default is number of CPUs on system> -v <verbose>
```

Comment:
You may want to break this snippet up across multiple lines to make it easier to read

```suggestion
python3 run_maki_on_compile_commands.py \
    -p <path to maki plugin> \
    -i <target program source directory> \
    -c <path to compile_commands.json> \
    -o <path to output maki analysis file, default is analysis.maki> \
    -j <number of threads, default is number of CPUs on system> \
    -v <verbose>
```